File: Sheet #1 (Data type - Conditions)/V_Comparison.cpp
#include <bits/stdc++.h>

using namespace std;
int main()
{
    int a, c;
    char b;
    cin >> a >> b >> c;

    if (b == '>')
    {
        if (a > c)
            cout << "Right";
        else
            cout << "Wrong";
    }
    else if (b == '<')
    {
        if (a < c)
            cout << "Right";
        else
            cout << "Wrong";
    }
    else if (b == '=')
    {
        if (a == c)
            cout << "Right";
        else
            cout << "Wrong";
    }

    return 0;
}

File: Sheet #1 (Data type - Conditions)/V_Comparison.java
import java.util.*;
import java.lang.*;
import java.io.*;

// The main method must be in a class named "Main"
public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int a = scan.nextInt();
        char b = scan.next().charAt(0);
        int c = scan.nextInt();

            if (b == '>')
            {
              if (a > c)
                System.out.println("Right");
              else
                System.out.println("Wrong");
            }
            else if (b == '<')
            {
            if (a < c)
              System.out.println("Right");
            else
              System.out.println("Wrong");
            }
            else if (b == '=')
            {
            if (a == c)
              System.out.println("Right");
            else
              System.out.println("Wrong");
              
    }
 
    }
}
